lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php

```diff
@@ -173,6 +173,11 @@ public function walkComparison(Comparison $comparison)
                     return false !== strpos(ClosureExpressionVisitor::getObjectFieldValue($object, $field), $value);
                 };
 
+            case Comparison::ICONTAINS:
+                return function ($object) use ($field, $value) {
+                    return false !== strpos(mb_strtolower(ClosureExpressionVisitor::getObjectFieldValue($object, $field)), mb_strtolower($value));
```

Comment:
Hmm, although it's correct, using mb_strtolower introduces dependency on ext-mbstring, which is not bundled in PHP. It's not hard dependency (required only when ICONTAINS is used), but it should probably be added to _suggest_ section in the composer.json.